src/core/js/router.js

```diff
@@ -345,13 +345,59 @@ define([
                 .attr('data-location', Adapt.location._currentLocation);
 
             this.setDocumentTitle();
+            this.setStartPageClass(type, id);
 
             // Trigger event when location changes.
             Adapt.trigger('router:location', Adapt.location);
 
             Adapt.wait.queue(onComplete);
         },
 
+        /**
+         * Sets the start page class as per configuration mentioned in "_start" attribute.
+         * Below code also considers all the example scenarios mentioned in below URL
+         * https://github.com/adaptlearning/adapt_framework/wiki/Content-starts-with-course.json#example-1
+         */
+        setStartPageClass: function(type, id) {
+            var startConfig = Adapt.course.get('_start');
+
+            if (!startConfig && !startConfig._isEnabled) return;
+
```

Comment:
This check should use || rather than &&. If startConfig is undefined, startConfig._isEnabled will throw an error.